### PR TITLE
feat/3964-library-components-tests

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

**Review by muhurov — APPROVED** [review]

File: src/modules/Library/features/Applet/Popups/RemoveAppletPopup/RemoveAppletPopup.test.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      />,
+      {
+        route,
+        routePath: route,
+      },
+    );
+
+    const confirmBtn = await waitFor(() => screen.getByText('Yes, Remove'));
+    fireEvent.click(confirmBtn);
+    expect(onCloseMock.mock.calls[0]);
```
Comment (alex-i-e) [inline src/modules/Library/features/Applet/Popups/RemoveAppletPopup/RemoveAppletPopup.test.tsx:32]: what did you check here, could you update it with more descriptive assertion?

**Review by nmialik — APPROVED** [review]

File: src/modules/Library/hooks/useReturnToLibraryPath.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
+import { auth } from 'modules/Auth';
 import { useEffect } from 'react';
 
-import { auth } from 'redux/modules';
+// import { auth } from 'redux/modules';
```
Comment (wildkiomi) [inline src/modules/Library/hooks/useReturnToLibraryPath.ts]: remove?

File: src/modules/Library/utils/getCheckedItems.test.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import { getFilteredSelectedItems, getUpdatedSelectedItems } from './getCheckedItems'; // Adjust the path to where the functions are actually located.
```
Comment (wildkiomi) [inline src/modules/Library/utils/getCheckedItems.test.ts]: it's TODO?

File: src/modules/Library/hooks/useReturnToLibraryPath.ts (hunk 2 above)
Comment (kdanilenka32) [reply src/modules/Library/hooks/useReturnToLibraryPath.ts]: fixed

File: src/modules/Library/utils/getCheckedItems.test.ts (hunk 3 above)
Comment (kdanilenka32) [reply src/modules/Library/utils/getCheckedItems.test.ts]: fixed

File: src/modules/Library/utils/dataFromStorage.test.ts [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+      itemNamePlusActivityName: 'Item1',
+      activityNamePlusId: 'Activity1ID',
+      activityName: 'Activity1',
+      activityKey: 'Key1',
+    },
+  ],
+};
+
+describe('Storage functions', () => {
+  beforeEach(() => {
```
Comment (wildkiomi) [inline src/modules/Library/utils/dataFromStorage.test.ts]: shouldn't you clear after all too?

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**